.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: Create Release
```

Comment:
Will this basically just create a draft for the tag, and when we publish the draft that means that we push it to PyPI?

Reply:
Correct, or we could just publish it right away 🤷

Reply:
I think both are OK. Maybe this is a good intermediary step, especially if you can inspect the build artifacts before publishing? We'd need to manually add the changelog anyway.

Reply:
We can use any bunch of tools to automate changelog. If we are tidy on PR and issue titles, that should be enough.

Reply:
You may want to add additional information so maybe it can pre-populate it with such a changelog? For 3.2 of Lektor we'd possibly state "This will be the last release with Python 2.7 support" on top of the change list.

Reply:
True, anyway :-)